src/lang/english.txt

```diff
@@ -1749,6 +1749,9 @@ STR_CONFIG_SETTING_REVERSE_AT_SIGNALS_HELPTEXT                  :Allow trains to
 
 STR_CONFIG_SETTING_QUERY_CAPTION                                :{WHITE}Change setting value
 
+STR_CONFIG_SETTING_COMPANY_AUTOPROTECT                          :Automatically protect new companies with pubkey
+STR_CONFIG_SETTING_COMPANY_AUTOPROTECT_HELPTEXT                 :Automatically protect new companies with cryptographic public key, allowing you to enter your companies without password.
```

Comment:
We need to ... "soften" the language here for sure :D But that will come when we do a proper review :)